src/main/java/mate/academy/service/Main.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.service;
+
+import java.math.BigDecimal;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
+public class Main {
+    private static final Injector injector
+            = Injector.getInstance("mate.academy.dao");
+
+    public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book book = new Book();
+        book.setTitle("Java Programming");
+        book.setPrice(BigDecimal.valueOf(29.99));
+
+        bookDao.create(book);
+        System.out.println("The book added: " + book);
+
+        Book foundBook = bookDao.findById(book.getId()).orElse(null);
```

Comment:
Consider handling the case where `findById` returns an empty `Optional` more explicitly, perhaps by logging a message or throwing an exception if a book is not found.